distribution/src/main/etc/bin/gmp-server-ctl.sh

```diff
@@ -12,22 +12,30 @@ set -u # Don't allow using non defined variables
 PAX_RUNNER_VERSION=${pax-runner.version}
 GMP_VERSION=${gmp.version}
 
+# Overrides locally JAVA_HOME to use java 8
+export JAVA_HOME=/usr/lib/jvm/java-1.8.0
```

Comment:
As we discussed we may want to allow a JAVA_HOME override from the environment and default to the system java8 if no variable is available